lambdatimes/src/components/Content/Tab.js

```diff
@@ -4,13 +4,23 @@ const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+      
+  let tabClass = '';
+  if (props.tab === props.selectedTab) {
+    tabClass = 'tab active-tab';
+  } else {
+    tabClass = 'tab';
+  }
   return (
     <div
-      className={''}
+      className={tabClass}
```

Comment:
Very good, look into the ternary operator, and think about how you could use that here.